src/include/connection.h

```diff
@@ -804,40 +802,41 @@ struct __wt_connection_impl {
     uint32_t server_flags;
 
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define WT_CONN_BACKUP_PARTIAL_RESTORE 0x00000001u
-#define WT_CONN_CACHE_CURSORS 0x00000002u
-#define WT_CONN_CACHE_POOL 0x00000004u
-#define WT_CONN_CALL_LOG_ENABLED 0x00000008u
-#define WT_CONN_CKPT_CLEANUP_RECLAIM_SPACE 0x00000010u
-#define WT_CONN_CKPT_GATHER 0x00000020u
-#define WT_CONN_CKPT_SYNC 0x00000040u
-#define WT_CONN_CLOSING 0x00000080u
-#define WT_CONN_CLOSING_CHECKPOINT 0x00000100u
-#define WT_CONN_CLOSING_NO_MORE_OPENS 0x00000200u
-#define WT_CONN_COMPATIBILITY 0x00000400u
-#define WT_CONN_DATA_CORRUPTION 0x00000800u
-#define WT_CONN_EVICTION_RUN 0x00001000u
-#define WT_CONN_HS_OPEN 0x00002000u
-#define WT_CONN_INCR_BACKUP 0x00004000u
-#define WT_CONN_IN_MEMORY 0x00008000u
-#define WT_CONN_LEAK_MEMORY 0x00010000u
-#define WT_CONN_LIVE_RESTORE_FS 0x00020000u
-#define WT_CONN_MINIMAL 0x00040000u
-#define WT_CONN_OPTRACK 0x00080000u
-#define WT_CONN_PANIC 0x00100000u
-#define WT_CONN_PREFETCH_RUN 0x00200000u
-#define WT_CONN_READONLY 0x00400000u
-#define WT_CONN_READY 0x00800000u
-#define WT_CONN_RECONFIGURING 0x01000000u
-#define WT_CONN_RECOVERING 0x02000000u
-#define WT_CONN_RECOVERING_METADATA 0x04000000u
-#define WT_CONN_RECOVERY_COMPLETE 0x08000000u
-#define WT_CONN_RTS_THREAD_RUN 0x10000000u
-#define WT_CONN_SALVAGE 0x20000000u
-#define WT_CONN_TIERED_FIRST_FLUSH 0x40000000u
-#define WT_CONN_WAS_BACKUP 0x80000000u
-    /* AUTOMATIC FLAG VALUE GENERATION STOP 32 */
-    wt_shared uint32_t flags_atomic;
+#define WT_CONN_BACKUP_PARTIAL_RESTORE 0x000000001ull
```

Comment:
Instead of increasing the flags size, how about moving a couple of them into `server_flags`? The `WT_CONN_EVICTION_RUN`, `WT_CONN_PREFETCH_RUN` and `WT_CONN_RTS_THREAD_RUN` are probable ones?